Jira.md

```diff
@@ -0,0 +1,29 @@
+# MetaBrainz bug tracking
+
+MetaBrainz uses Jira to track bugs. It is hosted at https://tickets.metabrainz.org/
+
+Each main MetaBrainz project has its own project in Jira.
+
+### Working on tickets
+
+If you want to work on a ticket, you don't need to ask for permission. Just check
+that the ticket hasn't been assigned to any other user, and assign it to yourself
+by clicking on the _Assign_ or _Assign to me_ link.
+
+If you want to work on a ticket which has been assigned to someone else, and it doesn't
+look like any work has been done for a while, you can post a question asking them if
+they are still going to complete the work.
+
+If you have any questions about a ticket, add a comment. Project leads will automatically
+get sent an email informing them of your comment, so you don't need to follow it up
+yourself.
+
+When you have assigned a ticket to yourself, set its status to in progress by clicking
+`Start Progress`. This shows others that you are actively working on a task.
+
+When you finish the task, open a Pull request on the relavent project. See [GitHub.md] with
+additional information about how to name and submit a Pull Request. In Jira, click `Resolve Issue`
```

Comment:
well. at least there's *only* 2... Do we need to bring this up at a meeting? I note that there are only 2 webapp projects on default. i wonder if they're just on default because no one changed them.

Can you post an image or text description of the MusicBrainz workflow? (from the MBS equivalent of https://tickets.metabrainz.org/plugins/servlet/project-config/AB/workflows).